etc/profile-m-z/npm.profile

```diff
@@ -0,0 +1,60 @@
+# Firejail profile for npm
+# Description: The Node.js Package Manager
+# This file is overwritten after every install/update
+# Persistent local customizations
+include npm.local
+# Persistent global definitions
+include globals.local
+
+blacklist /tmp/.X11-unix
+blacklist ${RUNUSER}
+
+noblacklist ${HOME}/.npm
+noblacklist ${HOME}/.npmrc
+
+noblacklist ${PATH}/bash
```

Comment:
Why only allow these from disable-shell.inc? IMHO it would make more sense to drop these individual paths and remove 'include disable-shell.inc' below.

Reply:
If just `/bin/sh` is needed?

Reply:
In my use of this profile, `/bin/sh` was insufficient, as some packages seemed to use `/bin/bash`.  The other shells do not seem to be used anywhere near as commonly for build scripts.